### ISSUE TITLE
My LTC wallet stays in offline modus

### ISSUE THREAD

**Issue by Brondhui** [pooler/electrum-ltc#81]
Hello Electrum,

Last week I bought some Lite Coins.
I have dowloaded the latest version of Electrum LTC 2.9.3.1
When entering the wallet it appairs to be offline all the time.
How can I change the status to online?

**Comment by pooler** [comment]
How long did you wait? The first time you run Electrum-LTC it has to download the blockchain headers, and that's currently a 100 MB download.

**Comment by Brondhui** [comment]
Hi Pooler,

I have the wallet for two weeks now and it still doesn’t work. I already Made two purchases of lite coins but can’t collect them in my wallet. I would also not be able to sell them

Regards,

Verstuurd vanaf mijn iPhone

Op 13 nov. 2017 om 14:02 heeft pooler <notifications@github.com<mailto:notifications@github.com>> het volgende geschreven:


How long did you wait? The first time you run Electrum-LTC it has to download the blockchain headers, and that's currently a 100 MB download.

**Comment by pooler** [comment]
Are you using a proxy?
Have you tried connecting to a different server?
Have you tried clearing the data directory (remember to do a backup first)?

**Comment by Brondhui** [comment]
With deleting the data directory you mean deleting and reinstalling the electrum wallet? I will try connecting to different a different server.

Verstuurd vanaf mijn iPhone

Op 13 nov. 2017 om 14:02 heeft pooler <notifications@github.com<mailto:notifications@github.com>> het volgende geschreven:


How long did you wait? The first time you run Electrum-LTC it has to download the blockchain headers, and that's currently a 100 MB download.

**Comment by pooler** [comment]
> With deleting the data directory you mean deleting and reinstalling the electrum wallet?

No, I mean deleting the directory containing Electrum-LTC's data. On Windows the default should be `\Users\YourUserName\AppData\Roaming\Electrum-LTC`. (Close Electrum-LTC and make a backup first.)

**Comment by Brondhui** [comment]
Where would I do that? I have 3 folders ( certs, forks, wallets) and 3 files Blockchain headers,condmfig and daemon)

Verstuurd vanaf mijn iPhone

Op 15 nov. 2017 om 09:58 heeft pooler <notifications@github.com<mailto:notifications@github.com>> het volgende geschreven:


With deleting the data directory you mean deleting and reinstalling the electrum wallet?

No, I mean deleting the directory containing Electrum-LTC's data. On Windows the default should be \Users\YourUserName\AppData\Roaming\Electrum-LTC. (Close Electrum-LTC and make a backup first.)

**Comment by Brondhui** [comment]
I Am not using a proxy user

Verstuurd vanaf mijn iPhone

Op 15 nov. 2017 om 09:58 heeft pooler <notifications@github.com<mailto:notifications@github.com>> het volgende geschreven:


With deleting the data directory you mean deleting and reinstalling the electrum wallet?

No, I mean deleting the directory containing Electrum-LTC's data. On Windows the default should be \Users\YourUserName\AppData\Roaming\Electrum-LTC. (Close Electrum-LTC and make a backup first.)

**Comment by pooler** [comment]
> Where would I do that? I have 3 folders ( certs, forks, wallets) and 3 files Blockchain headers,condmfig and daemon)

Backup and then delete the whole `Electrum-LTC` folder.

**Comment by baselab** [comment]
Hello,
I was having the same problem but then I figured out that I was behind a strict firewall.
I think Electrum needs to connect to servers with non-conventional TCP ports such as 9434,50002,50004,60002 etc.
I'm not aware of a server listening on statndard ports like 443, is there any?
